Dockerfile.gitlab

```diff
@@ -34,8 +39,11 @@ RUN opam pin add -yn current_docker.dev "./ocurrent" && \
     opam pin add -yn ocaml-version.dev "./ocaml-version" && \
     opam pin add -yn dockerfile.dev "./ocaml-dockerfile" && \
     opam pin add -yn dockerfile-opam.dev "./ocaml-dockerfile" && \
+    opam pin add -yn solver-service-api.dev "./solver-service" && \
+    opam pin add -yn solver-service.dev "./solver-service" && \
+    opam pin add -yn solver-worker.dev "./solver-service" && \
```

Comment:
```suggestion
```
Pinning the solver-worker is not required here.

Reply:
You're right, even `solver-service.dev` is not needed, only `solver-service_api.dev`